Sources/Client/ScoreboardView.cpp

```diff
@@ -49,17 +49,33 @@ namespace spades {
 		static const Vector4 spectatorTextColor = {220.f / 255, 220.f / 255, 0,
 		                                           1}; // Goldish yellow
 		static const auto spectatorTeamId = 255;       // Spectators have a team id of 255
+		
 
 		ScoreboardView::ScoreboardView(Client *client)
 		    : client(client), renderer(client->GetRenderer()) {
 			SPADES_MARK_FUNCTION();
-			ctf = NULL;
-			tc = NULL;
+			world = nullptr;
+			tc = nullptr;
+			ctf = nullptr;
+			image = nullptr;
+
+			// Use GUI font if spectator string has special chars
+			auto spectatorString = _TrN("Client", "Spectator{1}", "Spectators{1}", "", "");
+			auto has_special_char = 
+				std::find_if(spectatorString.begin(), spectatorString.end(),
+				[](char ch) {
+					ch = static_cast<unsigned char>(ch);
```

Comment:
Still, this line does _something_ that makes it work. Without it I get this when I have a font with non-ascii chars:
![image](https://cloud.githubusercontent.com/assets/6632271/21652714/89cf7ff6-d2b6-11e6-97fa-c2a707424b37.png)
And with it it works as intended.